app/scripts/modules/core/src/pipeline/service/execution.service.ts

```diff
@@ -22,7 +22,7 @@ export class ExecutionService {
   public get activeStatuses(): string[] {
     return ['RUNNING', 'SUSPENDED', 'PAUSED', 'NOT_STARTED'];
   }
-  private runningLimit = 30;
+  private runningLimit = SETTINGS.defaultExecutionResults || 30;
```

Comment:
It would be nice to add this to the `settings.ts` file, also, as an optional field — it helps people see what some of the different configuration options are without having to hunt through the code.

Reply:
Done!

Reply:
A better name for this would be `maxRunningExecutionsToRetrieve` or `backgroundRunningExecutionsLimit` — something to indicate what the setting will effect.